package.json

```diff
@@ -30,7 +30,8 @@
   ],
   "dependencies": {
     "clean-css": "4.1.9",
-    "gulp-util": "3.0.8",
+    "fancy-log": "^1.3.2",
```

Comment:
This should probably be a devDep (only used in gulpfile.js)

Reply:
Indeed! Good catch.

Reply:
We want to be careful about where dependencies go so plugin install size can be reduced as much as possible.

Reply:
@phated Nice catch, now that `plugin-error` lives on it's own I can definitely fix as you suggested. Will be incoming soon...

Reply:
fixed with [3.9.2](https://github.com/scniro/gulp-clean-css/releases/tag/3.9.2)